dart/lib/src/protocol/contexts.dart

```diff
@@ -123,7 +122,8 @@ class Contexts extends MapView<String, dynamic> {
 
   set gpu(SentryGpu? gpu) => this[SentryGpu.type] = gpu;
 
-  /// Trace context describes the GPU of the device.
+  /// looks like trace should not be SentrySpanContext but instead something simpler
+  /// which is not with all the fields from Span
   SentrySpanContext? get trace => this[SentrySpanContext.type];
```

Comment:
maybe we need a SentryTraceContext which only has the trace fields instead of the whole span context?

Reply:
makes sense to have its own SentryTraceContext since SentrySpanContext is polluted with fields that don't belong to the `trace` object